modules/server_new/src/main/scala/observe/server/keywords/ConditionOps.scala

```diff
@@ -101,8 +101,8 @@ object ConditionOps {
     def toPercentile(wv: Wavelength, elevation: Angle): Int = {
       // Pickering approximation (thanks Andy Stephens)
       val airmass         = 1.0 / sin(
-        (90.0 - elevation.toDoubleDegrees + 244.0 / (165.0 + 47.0 * pow(
```

Comment:
In my defense, the original formula has the expression `(90.0 - z)`. Then again, that `z` is not elevation, but zenith angle.